.eslintrc.json

```diff
@@ -5,13 +5,29 @@
     "standard",
     "prettier",
     "prettier/standard",
+    "plugin:unicorn/recommended",
     "plugin:ava/recommended",
     "plugin:you-dont-need-lodash-underscore/all"
   ],
   "rules": {
     "no-process-exit": 0,
     "object-shorthand": "error",
     "require-await": "error",
+    "unicorn/custom-error-definition": 2,
+    "unicorn/no-unused-properties": 2,
+    "unicorn/filename-case": [2, { "case": "kebabCase", "ignore": [".*.md"] }],
+    "unicorn/explicit-length-check": [2, { "non-zero": "not-equal" }],
+    "unicorn/no-process-exit": 0,
+    "unicorn/no-null": 0,
+    "unicorn/prevent-abbreviations": 0,
+    "unicorn/number-literal-case": 0,
+    "unicorn/expiring-todo-comments": 0,
+    "unicorn/no-fn-reference-in-iterator": 0,
+    "unicorn/no-unreadable-array-destructuring": 0,
+    "unicorn/import-style": 0,
+    "unicorn/prefer-optional-catch-binding": 0,
+    "unicorn/prefer-trim-start-end": 0,
+    "unicorn/no-reduce": 0,
```

Comment:
I agree with this one. [See rule documentation](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-reduce.md). However, this would require too much refactoring at this point.
I am wondering what your thoughts are on this?

Reply:
`reduce` can be hard to read most of the times. It might be too strict to forbid it completely.

Reply:
Should we lint against it in the future, and allow individual uses through `eslint-disable` comments?